pkg/kubeconfig/create_kubecfg.go

```diff
@@ -85,7 +85,8 @@ func BuildKubecfg(cluster *kops.Cluster, keyStore fi.Keystore, secretStore fi.Se
 
 	b.Context = clusterName
 
-	{
+	// add the CA Cert to the kubeconfig only if we didn't specify a SSL cert for the LB
+	if cluster.Spec.API.LoadBalancer.SSLCertificate == "" {
```

Comment:
Ah - this is giving us a panic in e2e... (e.g. https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/kops/5414/pull-kops-e2e-kubernetes-aws/6969/ )

I think you need:

```
if cluster.Spec.API == nil || cluster.Spec.API.LoadBalancer == nil || cluster.SPec.API.LoadBalancer.SSLCertificate == "" {
```

Reply:
Yes, I've seen the test a couple of days ago and yes I think that should be enough to fix, haven't had the time to do it do even if it's super quick, I was pretty busy these days... but the good news is: I will probably get it done tomorrow 😄